### PR TITLE
Add ngnix cluster resource

### PR DESCRIPTION
**Pull request by mpagot**
Add cluster resource to monitor the ngnix service, add it to the group with all the others.
Add a dedicated test to target the service directly.

### CODE REVIEW

File: azure/az_lb_http_pacemaker/azure_lb_cluster_test_01_crm_mv.sh [hunk 1]
```diff
@@ -67,13 +72,16 @@ test_step "Check ip address immediately after the move"
...
+# use curl to probe the webserver using the frontend IP
+# until the reply coe from the VM-02
 wait_for_takeover "${MYNAME}-vm-02"
 
 # Slow down, take a break, then check again, nothing should be changed.
 test_step "Check again later"
 sleep 30
 test_web "${MYNAME}-vm-02"
 
+# Repeate the same but this time from VM-02 to VM-01
```
Comment (lpalovsky) [inline azure/az_lb_http_pacemaker/azure_lb_cluster_test_01_crm_mv.sh]: s/Repeate/Repeat/

@coderabbitai Take this you looser AI. I have found a typo you have missed!

**Review by lpalovsky — APPROVED** [review]
LGTM

File: azure/az_lb_http_pacemaker/azure_lb_cluster_test_01_crm_mv.sh (hunk 1 above)
Comment (lpalovsky) [reply azure/az_lb_http_pacemaker/azure_lb_cluster_test_01_crm_mv.sh]: @coderabbitai Stop pretending to be nice. You cannot fool me, AI. I know what are you planning!

**Review by alvarocarvajald — APPROVED** [review]
LGTM

**Pull request merged**